pandas/tests/io/test_stata.py

```diff
@@ -361,16 +361,22 @@ def test_encoding(self, version):
 
         # GH 4626, proper encoding handling
         raw = read_stata(self.dta_encoding)
-        encoded = read_stata(self.dta_encoding, encoding="latin-1")
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
+            encoded = read_stata(self.dta_encoding, encoding='latin-1')
+            assert len(w) == 1
```

Comment:
(repost since GitHub hid it due to recent push): Hmmm...I was thinking that you use tm.assert_produces_warning or pytest.warns so that we explicitly ensure that the warning is produced.

Reply:
Yes, I think this can be:

```
with tm.assert_produces_warning(FutureWarning):
    encoded = read_stata(self.dta_encoding, encoding='latin-1')
```

(and the same for the other occurrence)